bibedit/record_fermilab.xml

```diff
@@ -0,0 +1,124 @@
+    <subfield code="c">VOLATILE:publication date YYYY-MM-DD</subfield>
+  </datafield>
+  <datafield tag="300" ind1=" " ind2=" ">
+    <subfield code="a">VOLATILE:999</subfield>
+  </datafield>
+  <datafield tag="520" ind1=" " ind2=" ">
+    <subfield code="a">VOLATILE:abstract</subfield>
+  </datafield>
+  <datafield tag="540" ind1=" " ind2=" ">
+    <subfield code="a">VOLATILE:terms of use, e.g CC License</subfield>
+    <subfield code="u">VOLATILE:url of file specifying access rights</subfield>
+  </datafield>
+  <datafield tag="542" ind1=" " ind2=" ">
+    <subfield code="f">VOLATILE:copyright statement</subfield>
+    <subfield code="u">VOLATILE:url of copyright statement</subfield>
+  </datafield>
+  <datafield tag="595" ind1=" " ind2=" ">
+    <subfield code="a">VOLATILE:hidden note</subfield>
+  </datafield>
+  <datafield tag="650" ind1="1" ind2="7">
+    <subfield code="a">VOLATILE:subject</subfield>
+    <subfield code="2">INSPIRE</subfield>
+  </datafield>
+  <datafield tag="653" ind1="1" ind2=" ">
+    <subfield code="a">VOLATILE:keyword</subfield>
+    <subfield code="9">VOLATILE:author</subfield>
+  </datafield>
+  <datafield tag="693" ind1=" " ind2=" ">
+    <subfield code="e">VOLATILE:experiment</subfield>
+  </datafield>
+  <datafield tag="695" ind1=" " ind2=" ">
+    <subfield code="a">VOLATILE:keyword</subfield>
+    <subfield code="2">VOLATILE:INSPIRE</subfield>
+  </datafield>
+  <datafield tag="700" ind1=" " ind2=" ">
+    <subfield code="a">VOLATILE:Familyname, Firstname</subfield>
+    <subfield code="j">VOLATILE:ORCID:</subfield>
+    <subfield code="m">VOLATILE:email</subfield>
+    <subfield code="u">VOLATILE:Affiliation</subfield>
+  </datafield>
+  <datafield tag="710" ind1=" " ind2=" ">
+    <subfield code="g">VOLATILE:collaboration</subfield>
+  </datafield>
+  <datafield tag="773" ind1=" " ind2=" ">
+    <subfield code="p">VOLATILE:journal</subfield>
+    <subfield code="v">VOLATILE:volume</subfield>
+    <subfield code="n">VOLATILE:issue</subfield>
+    <subfield code="c">VOLATILE:firstpage-lastpage</subfield>
+    <subfield code="m">VOLATILE:Erratum</subfield>
+    <subfield code="q">VOLATILE:conf acronym (if no CNUM exists)</subfield>
+    <subfield code="t">VOLATILE:name of conference (if no CNUM exists)</subfield>
+    <subfield code="w">VOLATILE:CNUM</subfield>
+    <subfield code="y">VOLATILE:year</subfield>
+  </datafield>
+  <datafield tag="856" ind1="4" ind2=" ">
+    <subfield code="u">VOLATILE:http: (for external link)</subfield>
+    <subfield code="y">VOLATILE:Fulltext</subfield>
+  </datafield>
+  <datafield tag="FFT" ind1=" " ind2=" ">
+    <subfield code="a">VOLATILE:http:</subfield>
+    <subfield code="d">VOLATILE:Fulltext</subfield>
+    <subfield code="t">VOLATILE:INSPIRE-PUBLIC</subfield>
+  </datafield>
+  <datafield tag="980" ind1=" " ind2=" ">
+    <subfield code="a">Fermilab</subfield>
+  </datafield>
+  <datafield tag="980" ind1=" " ind2=" ">
+    <subfield code="a">HEP</subfield>
```

Comment:
maybe I am confused, but isn't this template intended for those records which are not HEP ?